workers/tests/unittests/test_definitions.py

```diff
@@ -73,41 +54,6 @@ def test_get_raster_dataset_undefined(self):
         with self.assertRaises(RasterDatasetUndefinedError):
             definitions.get_raster_dataset("foo")
 
-    @mock.patch.dict("os.environ", {}, clear=True)
-    def test_get_data_dir_unset_env(self):
-        data_dir = definitions.get_data_dir()
-        expected = os.path.join(
-            os.path.dirname(
-                os.path.abspath(__file__),
-            ),
-            "..",
-            "..",
-            "..",
-            "data",
-        )
-        self.assertTrue(os.path.samefile(data_dir, expected))
-
-    def test_get_data_dir_set_env(self):
-        with tempfile.TemporaryDirectory() as tmpdirname:
-            custom_data_dir = os.path.join(tmpdirname, "oqt-custom")
-            os.mkdir(custom_data_dir)
-            with mock.patch.dict(
-                "os.environ",
-                {"OQT_DATA_DIR": custom_data_dir},
-                clear=True,
-            ):
-                data_dir = definitions.get_data_dir()
-            self.assertEqual(data_dir, custom_data_dir)
```

Comment:
I couldn't really find a replacement for this test. Is this not necessary anymore?